Sources/Plasma/Apps/plClient/CMakeLists.txt

```
@@ -243,6 +243,7 @@ target_link_libraries(
         CURL::libcurl
         "$<$<PLATFORM_ID:Darwin>:-framework Cocoa>"
         "$<$<PLATFORM_ID:Darwin>:-framework QuartzCore>"
+        "$<$<PLATFORM_ID:Darwin>:-framework IOKit>"
```

This line is already correct for non-macOS, I think. According to [the CMake docs for `CMAKE_SYSTEM_NAME`](https://cmake.org/cmake/help/v4.1/variable/CMAKE_SYSTEM_NAME.html#system-names-known-to-cmake) (which is what `PLATFORM_ID` corresponds to), `Darwin` actually means only macOS. For the other Apple OSes, `CMAKE_SYSTEM_NAME` is `iOS`, `tvOS`, etc.

If we want to be *really* correct, this generator expression would also have to check if we're targeting OS X older than 10.9, but that may be pointless complexity. (At least until Apple decides that linking against IOKit is verboten starting with macOS 29 or whatever.)

---

If we really care, I think this would work (untested). There's no generator expression for `CMAKE_OSX_DEPLOYMENT_TARGET`, so it has to be substituted in as a variable. This should be safe, because the value can't change depending on the build type or anything - only if the user re-runs CMake.

```suggestion
        "$<$<AND:$<PLATFORM_ID:Darwin>,$<VERSION_LESS:${CMAKE_OSX_DEPLOYMENT_TARGET},10.9>>:-framework IOKit>"
```